credmgr

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
I'm not sure whether to be impressed or horrified by the fact that you chose to use a shell script for this.